docs/application/dotnet/guides/security/privacy-related-permissions.md

```diff
@@ -17,6 +17,12 @@ The main features of the `Tizen.Security.PrivacyPrivilegeManager` class include:
 
 For a list of privacy-related privileges, see [Security and API Privileges](../../tutorials/sec-privileges.md).
 
+> **Note**
+>
+> Popups by `ppm_request_permission()` and `ppm_request_permissions()` are launched as 'group mode' with caller apps since Tizen 5.5.   
+> If the popup is terminated without full responses, all remained requests that aren't responded by the user will be treated as if the user selected `PRIVACY_PRIVILEGE_MANAGER_REQUEST_RESULT_DENY_ONCE` and you can request the permission again anytime you want.  
+> If you need to request multiple privileges then use `ppm_request_permissions()`.
```

Comment:
Another opinion to the last comment.

>Line 24:
>Slightly changed the structure keeping the original sentence intact.
>
>>In addition, user can request for multiple privileges using `ppm_request_permissions()`

In the note, `ppm_request_permissions()` is mentioned from the 1st line.
If `ppm_request_permissions()` is describe 1st time at this line, than "In addition" would be acceptable, as it has the nuance of describing additional option.

But as `ppm_request_permissions()` is already mentioned with `ppm_request_permission()`,
I think "in addition" here is not natural. This line is describing which one should be used among 2 options already presented to the reader.

| Original | Suggested |
| ---- | ---- |
| There is AA and AAs.<br>...<br>If you need to use plural, use AAs | There is AA and AAs.<br>...<br>In addition, user can use plural using AAs |

Another point is that the function name itself gives a hint to which one should be used for which case. Because of it, I'm not sure the effect of "In addition".

`ppm_request_permissions()` would be used for multiple privileges, rather then `ppm_request_permission()`.

So how about not using "In addition"? I'm not sure with that update is quite effective comparing to the original.
Please give your opinion.

Reply:
Modified the note a little bit. Please check the modified one.

Reply:
> 
> 
> Another opinion to the last comment.
> 
> > Line 24:
> > Slightly changed the structure keeping the original sentence intact.
> > > In addition, user can request for multiple privileges using `ppm_request_permissions()`
> 
> In the note, `ppm_request_permissions()` is mentioned from the 1st line.
> If `ppm_request_permissions()` is describe 1st time at this line, than "In addition" would be acceptable, as it has the nuance of describing additional option.
> 
> But as `ppm_request_permissions()` is already mentioned with `ppm_request_permission()`,
> I think "in addition" here is not natural. This line is describing which one should be used among 2 options already presented to the reader.
> Original 	Suggested
> There is AA and AAs.
> ...
> If you need to use plural, use AAs 	There is AA and AAs.
> ...
> In addition, user can use plural using AAs
> 
> Another point is that the function name itself gives a hint to which one should be used for which case. Because of it, I'm not sure the effect of "In addition".
> 
> `ppm_request_permissions()` would be used for multiple privileges, rather then `ppm_request_permission()`.
> 
> So how about not using "In addition"? I'm not sure with that update is quite effective comparing to the original.
> Please give your opinion.

@mijin85cho 
- In addition, is used for saying that something extra exists or is happening together with the thing that we are talking about. 

- FunctionX() can do this and in addition, Function() can do that also.
  I added **in addition** following this. 
- Also, function names are not always written keeping context in mind and I think irrespective of whether function name is singular or plural, the supporting text must be considered and written as per the situation. 
- What I intend to say is if I go by your opinion about the function `ppm_request_permissions()`, this function as per the name must always support be used for requesting multiple permission**s**, which is not true. 
- We definitely can also request for single permission that's why the author had to make it clear that we can request more than one permission too.

So I would like to know your opinion on this If you don't agree. We can go with:

The user can also request multiple privileges using ppm_request_permissions().

Can you please suggest a better rephrase, if you don't like the above rephrases.

Reply:
@nawab-samsung I still don't think the line is for something extra or happening together. To me, the line is just describing one of two functions. Perhaps that makes me to think "In addition" or "Also" too wordy for the simple sentence. 
Also I don't know why should we spend so much energy to this sentence while other issues to be checked are left.

> In addition, is used for saying that something extra exists or is happening together with the thing that we are talking about.

As the developer guide, I think the author's suggestion is not bad. How about keeping it?